plugin/index.js

```diff
@@ -38,6 +38,12 @@ const WebpackError = require('webpack/lib/WebpackError');
 class I18nOptimizerPlugin {
     apply(compiler) {
         compiler.hooks.compilation.tap(this.constructor.name, compilation => {
+            if (compiler.options.devtool && compiler.options.devtool.includes('eval')) {
```

Comment:
Do the plugin support all `devtool` value except `*eval*` ? Or does it support only `source-map` ? Because there's like 27 `devtool` possible values in webpack 5 and 13 in webpack 4. Just to be sure.

Reply:
I didn't test any of the others. I assume them to potentially work, but also mentioned `source-maps` here such that people having issues will hopefully try that.

Reply:
Just tested some right now and at least the ones I tested do work.